vllm/attention/backends/rocm_flash_attn.py

```diff
@@ -399,45 +407,39 @@ def forward(
         return output.view(num_tokens, hidden_size)
 
 
-def _naive_attention(
+def _sdpa_attention(
     query: torch.Tensor,
     key: torch.Tensor,
     value: torch.Tensor,
     seq_lens: List[int],
+    attn_metadata: ROCmFlashAttentionMetadata,
+    num_tokens: int,
+    num_heads: int,
+    head_size: int,
     scale: float,
 ) -> torch.Tensor:
-    output = torch.empty_like(query)
+    att_masks = [None] * len(seq_lens)
+    attn_metadata.attn_bias = att_masks
```

Comment:
The SDPA backend doesn't support ALiBi bias, right? If so, do we need these lines?

Reply:
good point. let me simplify this.

Reply:
@WoosukKwon Updated. Thanks for the careful review.